README.md

```diff
@@ -4,11 +4,10 @@ Rust SDK for Spice.ai
 
 ## Installation
 
-Add the following to your `Cargo.toml`:
```

Comment:
Why did we change this?  You don't add it to Cargo.toml?

Reply:
Since we recently published our sdk on crates.io we don't need to do this manually anymore: `crate add spiceai` updates `Cargo.toml`

https://crates.io/crates/spiceai